sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -3028,6 +3028,24 @@ class HiveDDLSuite
     }
   }
 
+  test("SPARK-36312: ParquetWriteSupport should check inner field") {
+    withView("v") {
+      spark.range(1).createTempView("v")
+      withTempPath { path =>
+        val e = intercept[AnalysisException] {
+          spark.sql(
+            """
+              | SELECT
```

Comment:
nit. Could you remove the redundant space here, @AngersZhuuuu ?

Reply:
Done